app/controllers/cart/base_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Cart::BaseController < ApplicationController
```

Comment:
WOOO! I'm so psyched how easily this worked. I still have a question about where authorization is actually occurring, but I think we can do a code review for this.

Reply:
Same here!